torch/csrc/autograd/python_variable.cpp

```diff
@@ -2148,8 +2150,12 @@ py::object torchDispatchFromTensorImpl(
   // NB: this may not be a python tensor if you got here from a mode!
   // TORCH_INTERNAL_ASSERT(isPythonTensor(self_t));
   append_overloaded_tensor(&overloaded_args, self_p.ptr());
-  auto args = py::reinterpret_steal<py::object>(PyTuple_New(1));
+  auto args = py::reinterpret_steal<py::object>(PyTuple_New(1 + (int)memory_format.has_value()));
   PyTuple_SET_ITEM(args.ptr(), 0, self_p.release().ptr());
+  if (memory_format.has_value()) {
+    auto memory_format_p = py::cast((int64_t)memory_format.value());
+    PyTuple_SET_ITEM(args.ptr(), 1, memory_format_p.ptr());
+  }
```

Comment:
This is a flagrant hack and should be marked as such.

Reply:
(the main trouble is that this won't compose if we ever get a second function that takes a different argument. No need to fix, but leave a breadcrumb for the future)